src/effects/RealtimeEffectStateUI.cpp

```diff
@@ -108,6 +108,9 @@ void RealtimeEffectStateUI::Show(AudacityProject& project)
          // project state is destroyed
          Hide(&project);
       });
+
+   mParameterChangedSubscription = mEffectUIHost->GetValidator()->Subscribe(
+      [this](auto) { ProjectHistory::Get(*mpProject).ModifyState(false); });
```

Comment:
This subscriber doesn't use the value inside the message.  Is there supposed to be another subscriber?

Reply:
Still unresolved question?

Reply:
I don't know what to say. I can make the structure empty if this will make you happier, I don't care